consensus/core/src/block_manager.rs

```diff
@@ -77,7 +82,16 @@ impl BlockManager {
         }
 
         accepted_blocks.sort_by_key(|b| b.reference());
-        Ok(accepted_blocks)
+
+        // Newly missed blocks
+        let missing_blocks_after = self
+            .missing_blocks
+            .difference(&missing_blocks_before)
```

Comment:
Can we leave a TODO to add logic to bound the computation here, or add argument that the computation is indeed bounded? When a node is behind or some peers are generating byzantine blocks, it seems `self.missing_blocks` can get arbitrarily large right now?

Reply:
Happy to leave a TODO, but this component will be refactored anyway to accommodate some byzantine resistance.